src/main/java/uk/gov/justice/digital/delius/service/CustodyService.java

```diff
@@ -48,6 +53,7 @@
 @Service
 @Slf4j
 public class CustodyService {
+    public static final String CONTEXT = "prison-to-probation";
```

Comment:
I this should be placed elsewhere if is needed but injected ${strings.key} .

Reply:
actually seems is used just locally (?) i case just change the visibility to private.

Reply:
ok